src/middlewared/middlewared/plugins/network.py

```diff
@@ -390,19 +387,23 @@ def query(self, filters, options):
             i['int_interface']: i
             for i in self.middleware.call_sync('datastore.query', 'network.interfaces')
         }
-        is_freenas = self.middleware.call_sync('system.is_freenas')
-        if not is_freenas:
+        licensed = self.middleware.call_sync('failover.licensed')
```

Comment:
We need to configure internal interfaces before a license is applied so we can contact the other node via middleware, otherwise the license cant be uploaded to the other node.

Think we should check for ENTERPRISE product type